### PR TITLE
Change Via to use via-host, disallowing comma (#24)

### PR DESCRIPTION
**Pull request by reschke**

### CODE REVIEW

**Comment by reschke** [comment]
Roy, could you please review the changed PR?

**Comment by royfielding** [comment]
I am thinking that we should reduce it to unreserved: ALPHA / DIGIT / "-" / "." / "_" / "~"

**Comment by reschke** [comment]
Hmmm, that's even more restrictive than token, which was used for pseudonym.

**Comment by reschke** [comment]
WFM. @mnot ?

File: draft-ietf-httpbis-semantics-latest.xml [hunk 1]
```diff
@@ -11868,6 +11868,15 @@ Content-Encoding: gzip
    <xref target="RFC4918" x:fmt="of" x:sec="11.2"/>) because of its general
    applicability (<xref target="status.422"/>).
 </t>
+<t>
+   The grammar definition for the Via field's "received-by" was unfortunately
```
Comment (mnot) [inline draft-ietf-httpbis-semantics-latest.xml]: "unfortunately" reads a little weird here. Not objecting strongly, but this para could be massaged a bit.

**Pull request merged**